packages/web/config/ibc-assets.ts

```diff
@@ -27,8 +27,8 @@ export const IBCAssetInfos: (IBCAsset & {
     counterpartyChainId: IS_TESTNET
       ? "axelar-testnet-lisbon-3"
       : "axelar-dojo-1",
-    sourceChannelId: IS_TESTNET ? "channel-312" : "channel-208",
-    destChannelId: IS_TESTNET ? "channel-22" : "channel-3",
+    sourceChannelId: IS_TESTNET ? "channel-135" : "channel-1946",
+    destChannelId: IS_TESTNET ? "channel-1946" : "channel-3",
```

Comment:
the testnet channel ids should be flipped for all 4 of these assets--source (osmosis side) as 1946 and dest (axelar side) as 135

Reply:
should be corrected now

Reply:
ok, I'll test it!

Reply:
@JeremyParish69  can we also add Axelar on the testnet? We should probably have a testnet for frontier as well @jonator .

Reply:
If anything, the testnet should be frontier, since it includes all assets from main. This would be simpler